_docker/drupal/drupal-filesystem/web/modules/custom/rhd_common/rhd_theme/composer.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "drupal/rhd_theme",
+  "type": "drupal-module",
+  "description": "Provides a theme negotiator to activate the new theme for applicable nodes",
+  "keywords": ["Drupal"],
+  "license": "GPL-2.0+",
+  "homepage": "https://www.drupal.org/project/rhd_theme",
+  "minimum-stability": "dev",
+  "support": {
+    "issues": "https://www.drupal.org/project/issues/rhd_theme",
+    "source": "http://cgit.drupalcode.org/rhd_theme"
```

Comment:
None of info here in support is actually correct.

Reply:
It was made by the generator tools, I just never updated. Not needed, file removed.